core/src/main/java/com/mapzen/android/graphics/MapView.java

```diff
@@ -49,8 +49,8 @@ public MapView(Context context, AttributeSet attrs) {
   }
 
   private void initDI(Context context) {
-    DI.init(context);
-    DI.component().inject(this);
+    CoreDI.init(context);
```

Comment:
We should replace `context` with `context.getApplicationContext()` when initializing dependency injection to prevent a memory leak. The `Context` associated with the `MapView` is tied to the `Activity` that created it.

I know we have been using the `MapView` context for this purpose for a while now but this just hit me when re-reading it.